agent/engine/docker_task_engine_test.go

```diff
@@ -63,6 +63,13 @@ const (
 	exitCode                    = 1
 	labelsTaskARN               = "arn:aws:ecs:us-east-1:012345678910:task/c09f0188-7f87-4b0f-bfc3-16296622b6fe"
 	taskSteadyStatePollInterval = time.Millisecond
+
+	// dockerVersionCheckDuringInit specifies if Docker client's Version()
+	// API needs to be mocked in engine tests
+	//
+	// isParallelPullCompatible is not invoked during engin intialization
+	// on windows. No need for mock Docker client's Version() call
+	dockerVersionCheckDuringInit = true
```

Comment:
In windows this value is false, wondering if windows will end up using the value here instead from `_windows_test.go` file.

Reply:
I've changed this to a var instead of const, given that we use different value for platforms - shadowing should handle it, although _windows_test.go itself doesn't actually reference it.